packages/eas-cli/src/graphql/mutations/BuildMutation.ts

```diff
@@ -119,6 +121,34 @@ export const BuildMutation = {
     );
     return nullthrows(data.build?.createIosBuild);
   },
+  async updateBuilMetadatadAsync(
+    graphqlClient: ExpoGraphqlClient,
+    input: {
+      buildId: string;
+      metadata: BuildMetadataInput;
+    }
+  ): Promise<BuildFragment> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .mutation<UpdateBuildMetadataMutation, UpdateBuildMetadataMutationVariables>(
+          gql`
+            mutation UpdateBuildMetadataMutation($buildId: ID!, $metadata: BuildMetadataInput!) {
+              build {
+                updateBuildMetadata(buildId: $buildId, metadata: $metadata) {
+                  id
+                  ...BuildFragment
+                }
+              }
+            }
+            ${print(BuildFragmentNode)}
+          `,
+          input,
+          { noRetry: true }
```

Comment:
Is this necessary?  I don't have an opinion, just asking.

Reply:
no, the update should be idempotent